README.md

```diff
@@ -183,6 +183,12 @@ I renamed the repository to "Awesome Alternatives in Rust". The original name wa
 
 * [gitui](https://github.com/extrawurst/gitui) - Blazing fast terminal-ui for git written in Rust 🦀
 
+### Performance
```

Comment:
Could you also add Performance to table of contents?

Reply:
Sure!

Reply:
Done